.github/workflows/ci.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        uses: purescript-contrib/setup-purescript@main
```

Comment:
I know by default it picks the latest, but I'd prefer for the compiler and spago versions to specified explicitly here.

It means we can control when to bump with an explicit PR and don't get prompted by a CI failure on an unrelated PR, and it means you have a chance of bisecting, because you know the tested compiler version for any given commit.